packages/sources/aleno/src/endpoint/price.ts

```diff
@@ -0,0 +1,47 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { TransportRoutes } from '@chainlink/external-adapter-framework/transports'
+import { config } from '../config'
+
+import { httpTransport } from '../transport/price-http'
+import { socketioTransport } from '../transport/price-socketio'
+
+export const inputParameters = new InputParameters(
```

Comment:
you should use `priceEndpointInputParametersDefinition` from `'@chainlink/external-adapter-framework/adapter'`